packages/graphics/src/Graphics.ts

```diff
@@ -613,34 +614,49 @@ export class Graphics extends Container
     /**
      * Specifies a simple one-color fill that subsequent calls to other Graphics methods
      * (such as lineTo() or drawCircle()) use when drawing.
-     * @param color - the color of the fill
-     * @param alpha - the alpha of the fill
+     * @param {PIXI.ColorSource} color - the color of the fill
+     * @param alpha - the alpha of the fill, will override the color's alpha
      * @returns - This Graphics object. Good for chaining method calls
      */
-    public beginFill(color = 0, alpha = 1): this
+    public beginFill(color: ColorSource = 0, alpha?: number): this
     {
         return this.beginTextureFill({ texture: Texture.WHITE, color, alpha });
     }
 
+    /**
+     * Normalize the color input from options for line style or fill
+     * @param {PIXI.IFillStyleOptions} options - Object object.
+     */
+    private normalizeColor<T extends IFillStyleOptions>(options: T): T
```

Comment:
passing an entire fill style object when only color and alpha is used feels a bit off.

Reply:
Yeah I will probably narrow the scope here to be `Pick<IFillStyleOptions, 'color'|'alpha'>` instead.